test/unix-socket.ts

```diff
@@ -31,10 +31,11 @@ if (process.platform !== 'win32') {
 	});
 
 	test('throws on invalid URL', async t => {
-		await t.throwsAsync(got('unix:', {retry: 0}), {
-			instanceOf: got.RequestError,
-			code: 'ENOTFOUND'
-		});
+		try {
+			await got('unix:', {retry: 0});
+		} catch (error) {
+			t.regex(error.code, /(ENOTFOUND|EAI_AGAIN)/);
```

Comment:
Why this change?

Reply:
Maybe the VM needs to be privileged? (run as root)?

Reply:
I think this test should be rewrote, `ENOTFOUND` is thrown because `unix:` is viewed as `http://unix/`

Reply:
Also no need to wrap it in parens

Reply:
> I think this test should be rewritten, ENOTFOUND is thrown because unix: is viewed as http://unix/

It shouldn't throw. If the hostname is `unix`, then Got does this:

https://github.com/sindresorhus/got/blob/0e0eefdd8538e910cba6d89df5a3268743aa7ec8/source/core/index.ts#L2311-L2324

Reply:
I think this needs more investigation